contracts/coordinator/src/contract/query.rs

```diff
@@ -94,3 +95,30 @@ pub fn chain_contracts_info(
     .change_context(Error::ChainContractsInfo)
     .map(ChainContractsResponse::from)
 }
+
+pub fn deployments(deps: Deps) -> Result<Vec<ChainContractsResponse>, Error> {
+    Ok(state::deployments(deps.storage)
+        .change_context(Error::ChainContractsInfo)?
+        .into_iter()
+        .map(|chains| ChainContractsResponse {
+            chain_name: chains.chain_name,
+            prover_address: chains.multisig_prover,
+            verifier_address: chains.voting_verifier,
+            gateway_address: chains.gateway,
+        })
+        .collect())
+}
+
+pub fn deployed_contracts(
```

Comment:
why do these functions have completely different names (both here and in state) when they effectively access the same data in both cases?